rankings/views.py

```diff
@@ -99,11 +100,19 @@ def get_context_data(self, **kwargs):
 
         group = get_object_or_404(Group, id=self.kwargs.get('group_pk', None))
         game = get_object_or_404(Game, id=self.kwargs.get('game_pk', None))
+        players = game.players.order_by('-ranking')
+        for player in players:
+            try:
+                rank_change = str(player.rankchange_set.get(game=game))
+            except ObjectDoesNotExist:
```

Comment:
You can change this to `except RankChange.DoesNotExist:` to be more specific and avoid the import of `ObjectDoesNotExist`. 

All models that extend form `ModelBase` get this `DoesNotExist` exception unless they are abstract.